controlplane/server/internal/registry/registry_sampler.go

```diff
@@ -237,7 +238,26 @@ func (sr *SamplerRegistry) UpdateSamplerStats(resource string, name string, coll
 			},
 		}
 
-		sampler = sr.createSampler(resource, name, []control.Tag{}, *initialConfig)
+		tags := []control.Tag{}
+
+		capabilities := control.Capabilities{
```

Comment:
same comment about the location of the default configuration for auto created samplers

Reply:
Changes performed as part of b236ffa562166626c55fab88659849f741bcf006